website/routes.py

```diff
@@ -349,6 +354,11 @@ def make_url_map(app):
         ),
     ])
 
+    process_rules(app, [
+        Rule('/explore/activity/popular/raw/', 'get', discovery_views.popular_activity_json, json_renderer),
```

Comment:
This route is unnecessary; as suggested in a previous comment, the new_and_noteworthy script can just make requests to piwik directly; no need to go through APIv1.